docs/tutorials/tutorial-sql-query-view.md

```diff
@@ -0,0 +1,168 @@
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+
+This tutorial demonstrates some useful features of the Query view in Apache Druid.
+
+You can use the Query view to ingest and query data. You can also use it to test and tune queries before you use them in API requests&mdash;for example, to perform [SQL-based ingestion](../multi-stage-query/api.md).
+
+The tutorial guides you through the steps to ingest sample data, and query the ingested data using some Query view features.
+
+## Prerequisites
+
+Before you follow the steps in this tutorial, download Druid as described in the [quickstart](./index.md) using the [micro-quickstart](../operations/single-server.md#micro-quickstart-4-cpu-16gib-ram) single-machine configuration and have it running on your local machine. You don't need to have loaded any data.
+
+## Run a demo query to ingest data
+
+In this section you load the demo queries that are included in Druid, and run a SQL task to ingest sample data into a datasource.
+
+1. Navigate to the Druid console at [http://localhost:8888](http://localhost:8888) and click **Query**.
+
+2. Click the ellipsis icon at the bottom of the query window and select **Load demo queries**. Note that loading the demo queries replaces all of your current query tabs. The demo queries load in several query tabs:
+
+   ![demo queries](../assets/tutorial-sql-demo-queries.png)
+
+3. Click the **Demo 1** tab. This query ingests sample data into a datasource called **kttm_simple**. Click the **Demo 1** tab heading again and note the options&mdash;you can rename, copy and duplicate tabs.
+
+4. Click **Run** to ingest the data.
+
+5. When ingestion is complete, Druid displays the time it took to complete the insert query, and the new datasource **kttm_simple** displays in the left pane.
+
+In this tutorial you only use the **Demo 1** query. You can explore the other tabs to see example queries that ingest, transform, and query sample data.
+
+## View and filter query results
+
+In this section you run some queries against the new datasource and perform some operations on the query results.
+
+1. Click **+** to the right of the existing tabs to open a new query tab.
+
+2. Click the name of the datasource **kttm_simple** in the left pane to display some automatically generated queries:
+
+   ![auto queries](../assets/tutorial-sql-auto-queries.png)
+
+3. Click **SELECT * FROM kttm_simple** and run the query.
+
+4. In the query results pane, click **Chrome** anywhere it appears in the **browser** column then click **Filter on: browser = 'Chrome'** to filter the results.
+
+In this tutorial you only use the **Demo 1** query. You can explore the other tabs to see examples of queries that ingest, transform, and query sample data.
+
+## Run aggregate queries
+
+In this section you run some aggregate queries and perform some operations on the query results.
```

Comment:
Do we need some context about why aggregation is good in Druid? A link out aggregation functions?

Reply:
Updated.